neqo-client/src/main.rs

```diff
@@ -832,7 +832,7 @@ fn handle_test(
     local_addr: SocketAddr,
     remote_addr: SocketAddr,
     hostname: &str,
-    urls: &[Url],
+    urls: Vec<Url>,
```

Comment:
I would prefer if you changed this to a VecDeque.  You can see that this argument is transformed into a VecDeque for processing anyway, so you might as well save a transformation.

(At first, this change looked to be gratuitous.  The change works without it.  But I think that you are right to change to a pass-by-value in this case as the test functions all basically rely on having ownership of the list of URLs.)

Reply:
I missed this. Thank you. 

ad02584d